### DISCUSSION TITLE
Production throws TypeError: e is null

### DISCUSSION THREAD

**Discussion by tintin10q** [nuxt/nuxt#17137] [Questions]
Hi,

My website works fine in development mode but then when I build I get:

`[Vue warn]: [@vue/server-renderer] Invalid VNode type: Symbol(Fragment) (symbol)` in the console 

and in the browser I get:

```TypeError: e is null
    NuxtJS 62
        nextSibling
        m
        qe
        pe
        y
        N
        run
        update
        re
        U
        y
        K
        Z
        y
        N
        run
        update
        Ht
        Ca
        promise callback*_a
        Di
        effect
        vs
        Jo
        xt
        aa
        set value
        l
        Bb
        Ht
        lt
        n
        Td
        kd
        $d
        k
        $
        y
        N
        run
        update
        he
        Pe
        U
        y
        N
        run
        update
        he
        Pe
        U
        y
        I
        k
        $
        y
        I
        k
        $
        y
        N
        run
entry-8a697c9a.mjs:1:12607
```

multiple times with different lines. 

For instance there is also this one:
```TypeError: e is null
    NuxtJS 15
        parentNode
        N
        run
        update
        re
        U
        y
        K
        Z
        y
        N
        run
        update
        Ht
        Ca
entry-8a697c9a.mjs:1:12607
    NuxtJS 4```

**Comment by Flanker72** [comment]
There may be a plenty of cases that causes such error.
Try to check these:
- you have no plain text mixed with tags in the same node (`<template>123<div>456</div>789</template>`)
- you haven't initialized the additional Vue instance somehow

**Reply by tintin10q** [reply]
What do you mean with `you have no plain text mixed with tags in the same node` is the example you gave a good or bad thing to have?

I don't know what you mean with `you haven't initialized the additional Vue instance somehow`

Why would this issue only appear in production mode?

**Reply by Flanker72** [reply]
Hi!
1. Given example is bad. You should avoid mixing nodes like this.
2. Such error appears frequently when there is some Vue instances on the page. This may be when you inject VueJS script from outside in addition to Nuxt default script. This is a very strange situation, however I must say about this problem here.
3. IDK, sorry(
